chapters/usage-profile.md

```diff
@@ -0,0 +1,326 @@
+</UsageScope>
+```
+
+
+## 31.3 UsageScope SPDX identifier field <a name="31.3"></a>
+
+### 31.3.1 Description
+
+Uniquely identify any element in an SPDX document which may be referenced by other elements. These may be referenced internally and externally with the addition of the SPDX document identifier. The metadata for the SPDX identifier field is shown in Table NN.
+
+**Table 31.3 — Metadata for the UsageScope SPDX identifier field**
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | No |
+| Cardinality | 1..1 |
+| Format | "SPDXRef-"`[idstring]` <br> where `[idstring]` is a unique string containing letters, numbers, `.`, and/or `-`. |
+
+### 31.3.2 Intent
+
+There may be several cases of the same package within an SPDX document. Each element needs to be able to be referred to uniquely so that relationships between elements can be clearly articulated.
+
+### 31.3.3 Examples
+
+EXAMPLE 1 Tag: `UsageScopeSPDXID:`
+
+```text
+UsageScopeSPDXID: SPDXRef-1
+```
+
+EXAMPLE 2 RDF: The URI for the element will follow the form:
+
+```text
+[SPDX document namespace]#[SPDX identifier]
+```
+
+See [6.5](document-creation-information.md#6.5) for the definition of the SPDX document namespace and [6.3](document-creation-information.md#6.3) for the definition of the SPDX identifier
+
+Using `xml:base`:
+
+```text
+<rdf:RDF xml:base="http://acme.com/spdxdocs/acmeproj/v1.2/1BE2A4FF-5F1A-48D3-8483-28A9B0349A1B">
+    ...
+    <UsageScope rdf:about="#SPDXRef-1">
+    ...
+    </UsageScope>
+</rdf:RDF>
+```
+
+Using document URI:
+
+```text
+<UsageScope rdf:about="http://acme.com/spdxdocs/acmeproj/v1.2/1BE2A4FF-5F1A-48D3-8483-28A9B0349A1B#SPDXRef-1">
+    ...
+</UsageScope>
+```
+
+## 31.4 Package Release Date field <a name="31.4"></a>
+
+### 31.4.1 Description
+
+This field provides a place for recording package release date
+
+Table 31.4 — Metadata for the package release date
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | No |
+| Cardinality | 0..1 |
+| Format | `YYYY-MM-DDThh:mm:ssZ`<br>where:<br><ul><li>`YYYY` is year</li><li>`MM` is month with leading zero</li><li>`DD` is day with leading zero</li><li>`T` is delimiter for time</li><li>`hh` is hours with leading zero in 24 hour time</li><li>`mm` is minutes with leading zero</li><li>`ss` is seconds with leading zero</li><li>`Z` is universal time indicator</li></ul> |
+
+
+### 31.4.2 Intent
+
+Description of the package release date for strict identification of the prerequisite assumptions of usage scope by the package supplier.
+
+### 31.4.3 Examples
+
+EXAMPLE 1 Tag: `PackageReleaseDate:`
+
+```text
+PackageReleaseDate: 2010-01-29T18:30:22Z
+```
+
+EXAMPLE 2 RDF: Property `spdx:packageReleseDate` in class `spdx:UsageScope`
+
+```text
+<UsageScope rdf:about="...">
+    <packageReleaseDate> 2010-01-29T18:30:22Z </packageReleaseDate>
+</UsageScope>
+```
+
+## 31.5 Package Built Date field <a name="31.5"></a>
```

Comment:
Same comment here as for the Package Release Date field above. This appears to be recording objective metadata about the Package, and seems like it should go in the Package Information section.